Comp. Sci./SQL/Database Clients/what-is-a-database-client.md

```diff
@@ -0,0 +1,47 @@
+# What is a database client?
+author: Nick Daminov
+
+levels:
+
+  - beginner
+
+  - basic
+
+type: normal
+
+category: must-know
+
+links:
+
+  - '[Top 10 free Database Management Tools](https://techtalk.gfi.com/top-10-free-database-tools-for-sys-admins/)'
+  - '[Comparison of those tools](https://en.wikipedia.org/wiki/Comparison_of_database_tools)'
+
+---
+## Content
+
+Database Client is a piece of software (also known as Database Management Tool) that gets information from the Database Server which holds all the data. Database Client is essential for all the computers that want to have access to the database.
+
+As a user we do not need to know how out data gets allocated on the storage device and how the server configuration gets set up. So let's forget the backend part (server) of using a database and look at Database Client.
+
+Some of the Database Clients can act as one or both of the following things:
+ - The end user GUI/command line database management system which is connected to the database server.
+ - Intermediate level between other users' programs which require data and the database server.
+
+---
+## Practice
+
+Is the Database Client software essential the computer to have access to the data?
+???
+
```

Comment:
1. isnt the data stored on a server?
2. we cant make it look like to separate questions, it will be like

Which place is the data in a client-server model stored?
???

Who issues the queries, and who receives them?

The ??? issues queries, and the ??? receives them.

- answer1
- answer2
- answer3
- answer4
- answer5